collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
@MarkRx @tabladrum 
While testing the latest changes, I was comparing the jira and versionone collectors and noted some differences in the way the data is being collected and shown in the UI.
- No collector items get created when the jira collector is run as opposed to versionone.
- The components collection that is created when you save the jira config doesn't have the corresponding collector_item
Components
/* 1 */
{
    "_id" : ObjectId("588187ebf027536d790834a4"),
    "_class" : "com.capitalone.dashboard.model.Component",
    "name" : "DashAppName",
    "collectorItems" : {}
}
- The widget inside the Dashboard collection is being tied directly to the Collector via "collectorIdOverride" : "58818744f027536d1b592e96". In VersionOne we tie the widget to the component which in turn ties to the Collector.

Should we be consistent in the jira / versionone designs ? It will also be easier to refactor in the future.